### DISCUSSION TITLE
How much time does Envato take to approve one HTML item?

### DISCUSSION THREAD

**Discussion by MdRaisulIslam02** [community/community#134047] [Programming Help] [answered]
### Body

I uploaded an HTML template on ThemeForest. I want to know how much time it will take to review my item.

Thanks

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MdRasel5** [comment, marked as answer]
This may take 5-7 days.
